contents/javascript/Javascript-framework-Redux.md

```diff
@@ -0,0 +1,231 @@
+<frontmatter>
+  title: Redux
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 2
+</frontmatter>
+
+<div class="website-content">
+  
+{{ booktitle | safe }}
+
```

Comment:
Seems like this is somewhat tied to React still, perhaps mention it as a reader assumption and link to the react article?

Reply:
Is it really, how so? 😞

Reply:
I think its fine now that the react quote is in the popover.
Don't think its necessarily a bad thing in this case as well; Having some knowledge of front end frameworks can lead to a better understanding of this article (redux) in general